ui/admin/tests/integration/components/worker-diagram/index-test.js

```diff
@@ -3,39 +3,49 @@ import { setupRenderingTest } from 'admin/tests/helpers';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import { setupIntl } from 'ember-intl/test-support';
+import {
+  disableFeature,
+  enableFeature,
+} from '../../../helpers/features-service';
 
 module('Integration | Component | worker-diagram/index', function (hooks) {
   setupRenderingTest(hooks);
   setupIntl(hooks);
 
-  let featuresService;
+  // let featuresService;
   const targetWorkerFilterIngress = 'target-worker-filters-v2-ingress';
   const targetWorkerFilterHCP = 'target-worker-filters-v2-hcp';
 
   test('it renders a single filter diagram', async function (assert) {
-    featuresService = this.owner.lookup('service:features');
-    featuresService.disable(targetWorkerFilterIngress);
-    featuresService.disable(targetWorkerFilterHCP);
+    // featuresService = this.owner.lookup('service:features');
+    // featuresService.disable(targetWorkerFilterIngress);
+    // featuresService.disable(targetWorkerFilterHCP);
+    disableFeature(targetWorkerFilterIngress);
+    disableFeature(targetWorkerFilterHCP);
     assert.expect(1);
     await render(hbs`<WorkerDiagram />`);
 
     assert.dom('[data-test-single-filter-egress-off]').isVisible();
   });
 
   test('it renders a dual filter diagram when `target-worker-filters-v2-ingress` is enabled', async function (assert) {
-    featuresService = this.owner.lookup('service:features');
-    featuresService.enable(targetWorkerFilterIngress);
-    featuresService.disable(targetWorkerFilterHCP);
+    // featuresService = this.owner.lookup('service:features');
+    // featuresService.enable(targetWorkerFilterIngress);
+    // featuresService.disable(targetWorkerFilterHCP);
+    enableFeature(targetWorkerFilterIngress);
+    disableFeature(targetWorkerFilterHCP);
     assert.expect(1);
     await render(hbs`<WorkerDiagram />`);
 
     assert.dom('[data-test-dual-filter-egress-off-ingress-off]').isVisible();
   });
 
   test('it renders a HCP dual filter diagram when `target-worker-filters-v2-hcp` is enabled', async function (assert) {
-    featuresService = this.owner.lookup('service:features');
-    featuresService.enable(targetWorkerFilterIngress);
-    featuresService.enable(targetWorkerFilterHCP);
+    // featuresService = this.owner.lookup('service:features');
```

Comment:
Could we delete the commented old code?
My understanding is that we are leaving this comments as a backup if something fails? We can always look up git history 😃

Reply:
Ooops. I thought I got them all. I will remove these before merging.

Reply:
lol being there done that :sweat_smile: